app-tracking-protection/vpn-store/src/main/java/com/duckduckgo/mobile/android/vpn/dao/VpnAppTrackerBlockingDao.kt

```diff
@@ -87,9 +87,6 @@ interface VpnAppTrackerBlockingDao {
     @Query("SELECT * from vpn_app_tracker_exclusion_list")
     fun getAppExclusionListFlow(): Flow<List<AppTrackerExcludedPackage>>
 
-    @Insert
-    fun setExclusionListMetadata(appTrackerExclusionListMetadata: AppTrackerExclusionListMetadata)
```

Comment:
To keep the PR smaller and safer, I did not fully remove all metadata references. Do we think it's ok or should I remove the relevant tables and references? Or do we want to implement a similar mechanism in the PrivacyConfig?